Services/ExamGenerator.Services.Data/QuestionTypeService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using ExamGenerator.Data.Common.Repositories;
+using ExamGenerator.Data.Models;
+using ExamGenerator.Web.InputModels.QuestionType;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExamGenerator.Services.Data
+{
+    public class QuestionTypeService:IQuestionTypeService
+    {
+        private readonly IDeletableEntityRepository<QuestionType> repository;
+
+        public QuestionTypeService(IDeletableEntityRepository<QuestionType> repository)
+        {
+            this.repository = repository;
+        }
+
+        public async Task Add(QuestionTypeInputModel model)
+        {
+            var questionType = new QuestionType();
+            questionType.Type = model.Type;
+            await repository.AddAsync(questionType);
```

Comment:
Use model.To<QuestionType>();
Have a look in SubjectService, Add method for more details